pkg/pillar/zedcloud/send.go

```diff
@@ -117,7 +117,7 @@ func SendOnAllIntf(ctx *ZedCloudContext, url string, reqlen int64, b *bytes.Buff
 			switch resp.StatusCode {
 			case http.StatusServiceUnavailable:
 				senderStatus = types.SenderStatusUpgrade
-			case http.StatusNotFound:
+			case http.StatusNotFound, http.StatusBadRequest:
```

Comment:
We get BadRequest from some controllers due to internal errors so I don't think we should treat that as a 404.

Reply:
But I see code 400 from controller. And for now we do uuid update per config respond and I can see 400 here https://github.com/lf-edge/eve/blob/master/api/APIv2.md#configuration.

Reply:
Very odd, the APIv2.md has 400 in lots of places where I expect to see a 404. @deitch I think you wrote the original API.md - any comments?